chia/util/streamable.py

```diff
@@ -44,6 +45,35 @@ def __init__(self, message: str, cls: Type[object]):
         )
 
 
+class ParameterMissingError(StreamableError):
+    def __init__(self, cls: type, missing: List[str]):
+        super().__init__(
+            f"{len(missing)} field{'s' if len(missing) > 0 else ''} missing for {cls.__name__}: {', '.join(missing)}"
+        )
+
+
+class InvalidTypeError(StreamableError):
+    def __init__(self, expected: type, actual: type):
+        super().__init__(
+            f"Invalid type: Expected {expected.__name__}, Actual: {actual.__name__}",
+        )
+
+
+class InvalidSizeError(StreamableError):
+    def __init__(self, expected: int, actual: int):
+        super().__init__(
+            f"Invalid size: Expected {expected}, Actual: {actual}",
+        )
+
+
+class ConversionError(StreamableError):
+    def __init__(self, value: Any, to_type: type, exception: Exception):
```

Comment:
Does this need to be `Any` vs. `object`?

Reply:
6aced9cc55c5e22d20356566b1871f1cee5d10c7, nope :)